.github/workflows/hds-demo-preview.yml

```diff
@@ -0,0 +1,124 @@
+name: hds-demo-preview
+
+on:
+  pull_request:
+    branches:
+      - development
+      - release-*
+  push:
+    branches:
+      - development
+    
+
+jobs:
+ build_and_publish_demo:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+
+    env:
+      PATH_PREFIX: "/hds-demo/preview_${{ github.event.number != '' && github.event.number || github.ref_name }}"
+      DEMO_NAME: preview_${{ github.event.number != '' && github.event.number || github.ref_name }}
+
+    steps:
+      - name: Checkout code hds
+        uses: actions/checkout@v4
+
+      - name: Read .nvmrc
+        run: echo "NODE_VERSION=$(cat .nvmrc)" >> $GITHUB_OUTPUT
+        id: nvmrc
+
+      - name: setup node ${{ steps.nvmrc.outputs.NODE_VERSION }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: '${{ steps.nvmrc.outputs.NODE_VERSION }}'
+          registry-url: 'https://registry.npmjs.org'
+
+      # Github cache
+      - name: get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
+        
+      - name: restore yarn cache
+        uses: actions/cache@v3
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: yarn-cache-folder-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            yarn-cache-folder-
+
+      - name: restore lerna
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: yarn-node_modules-folder-${{ hashFiles('**/yarn.lock') }}
+
+      # Build
+      - name: install dependencies
+        run: |
+          yarn config set network-timeout 300000 
+          yarn
+
+      # Build site
+      - name: build design tokens package
+        run: yarn build
+        working-directory: ./packages/design-tokens
+
+      - name: build core package
+        run: yarn build
+        working-directory: ./packages/core
+
+      - name: build react package
+        run: yarn build
+        working-directory: ./packages/react
+
+      - name: build hds-js package
+        run: yarn build:hds-js
+        working-directory: ./packages/react
+
+      - name: build site package
+        run: yarn build --prefix-paths
+        working-directory: ./site
+
+      # Build core storybook
+      - name: build core storybook
+        run: yarn build-storybook
+        working-directory: ./packages/core
+
+      - name: move core storybook under site
```

Comment:
should core storybook go under ./site/public/storybook/core?

Reply:
This is copied from gh-pages.yml. 

This is how it has done on 
https://hds.hel.fi/storybook/core

Reply:
ok maybe I just missed something myself :D Monday morning :D